yarn-project/types/src/noir/index.ts

```diff
@@ -40,10 +42,10 @@ export interface NoirFunctionAbi {
 export interface NoirFunctionEntry {
   /** The name of the function. */
   name: string;
-  /** The type of the function. */
-  function_type: NoirFunctionType;
-  /** Whether the function is internal. */
-  is_internal: boolean;
+  /** Whether the function is unconstrained. */
+  is_unconstrained: boolean;
+  /** Attributes injected via macros. */
```

Comment:
```suggestion
  /** Custom attributes attached to function */
```

These aren't injected by the macros